_includes/syllabus.html

```diff
@@ -1,16 +1,21 @@
 {% comment %}
   Display syllabus in tabular form.
   Days are displayed if at least one episode has 'start = true'.
+  Days are displayed as Parts if at least one episode has 'part = true'.
 {% endcomment %}
 <div class="syllabus">
   <h2 id="schedule">Schedule</h2>
 
   {% assign lesson_number = 0 %}
   {% assign day = 0 %}
   {% assign multiday = false %}
+  {% assign partnotday = false %}
   {% for episode in site.episodes %}
     {% if episode.start %}{% assign multiday = true %}{% break %}{% endif %}
   {% endfor %}
+  {% for episode in site.episodes %}
```

Comment:
You can avoid this look by including the next line in the previous loop. This should make the compilation faster.